config/cmake/modules/FindCaliper.cmake

```
@@ -16,7 +16,28 @@
 
 include(MfemCmakeUtilities)
 mfem_find_package(Caliper CALIPER CALIPER_DIR
-       	"include" "caliper/cali.h"
-       	"lib" "caliper"
-  	"Paths to headers required by Caliper."
-       	"Libraries required by Caliper.")
+      "include" "caliper/cali.h"
+      "lib" "caliper"
+      "Paths to headers required by Caliper."
+      "Libraries required by Caliper.")
+
+# Append adiak path/lib if the user provided ADIAK_DIR
+if(ADIAK_DIR AND EXISTS ${ADIAK_DIR})
+    list(APPEND CALIPER_INCLUDE_DIRS ${ADIAK_DIR}/include)
+    
+    unset(_lib)
+    find_library(_lib "adiak" NO_CACHE PATHS ${ADIAK_DIR}/lib)
+    list(APPEND CALIPER_LIBRARIES ${_lib})
```

With the above suggestion, this should become:
```suggestion
    list(APPEND CALIPER_LIBRARIES ${adiak_LIBRARIES})
```

---

Done